### ISSUE TITLE
Dalaran Heist: Special music doesn't play, while spectating someone fight final boss

### ISSUE THREAD

**Issue by Adusak90** [HearthSim/hs-bugs#1194]
Other adventures like Dungeon run, Witchwood or Icecrown citadel works properly by playing different music in the same conditions.

**Comment by Jetz72** [comment]
Any more reports to corroborate this? Did you check multiple Dalaran final bosses? Which platform? Mobile platforms treat adventure music different, sometimes not downloading it to save on file size. Do you own the adventure? 

I've watched someone fight a final boss, but I didn't pay attention to the music.

**Comment by Adusak90** [comment]
Yup, I watched a lot of Dalaran heist lately, and I never actually realised that they were facing the final boss because of lack of music (only way to tell was starting hp, if it was in middle of fight I had no idea about it and was informed after the fact when they started new run).

**Comment by Jetz72** [comment]
Confirmed this with Trogg while testing other bugs. I attempted to screenshot it but then realized the problem with that.

The music I was hearing was this: 
https://www.youtube.com/watch?v=Rmi03oHRgqc
